PluginGenerator/DataModel/DebtRemediationFunctionType.cs

```diff
@@ -0,0 +1,30 @@
+﻿/*
+ * SonarQube Roslyn SDK
+ * Copyright (C) 2015-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarQube.Plugins.Roslyn
+{
+    public enum DebtRemediationFunctionType
```

Comment:
SonarC# and SonarVB currently support only `CONSTANT_ISSUE` so I am not sure if we will be able to provide the other values.

Reply:
I'll add a comment saying they are unsupported by the C# plugin currently.